### ISSUE TITLE
Question about agent.eval_mode()

### ISSUE THREAD

**Issue by xylee95** [pfnet/pfrl#153]
How do I switch back the agent to training mode after using `with agent.eval_mode()`?

I'm following the quick-start example and I want to evaluate my agent intermittently during training, rather than at the end of the training. What is the proper way of returning the agent to training setting after running a certain number of episodes of evaluation?

 I'm writing my own custom training loop rather than using the pre-defined experiments function.

**Comment by muupan** [comment]
The agent will switch back to the original mode automatically once it leaves the `with agent.eval_mode()` context.
```python
print("training", agent.training)
with agent.eval_mode():
    print("training", agent.training)
print("training", agent.training)
```
will print
```
training True
training False
training True
```

**Comment by xylee95** [comment]
Great, thank you for clarifying!
